src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+  - Anything that service might do to handle the request (_S_)
+  
+_Note:_ Starred spans represent spans that are the parent of a later transaction (and its root span).
+  
+### Tracing Data Model
 
-Each [trace]({%- link _documentation/performance/performance-glossary.md -%}#trace) has a marker called a `trace_id`. Trace IDs are pseudorandom fixed-length alphanumeric character sequences.
+> "Show me your flowchart and conceal your tables, and I shall continue to be mystified. Show me your tables, and I won't usually need your flowchart; it'll be obvious." 
+>
+> -- [Fred Brooks](https://en.wikipedia.org/wiki/Fred_Brooks), The Mythical Man Month (1975)
 
-By collecting traces of your users as they use your applications, you can begin to reveal some insights such as:
+While the theory is interesting, ultimately any data structure is defined by the kind of data it contains, and relationships between data structures are defined by how links between them are recorded. Traces, transactions, and spans are no different.
 
-- What occurred for a specific error event, or issue
-- What conditions are causing bottlenecks or latency issues in the application
-- Which endpoints or operations consume the most time
+**Traces**
 
-### Trace Propagation Model
+Traces are not an entity in and of themselves. Rather, a trace is defined as the collection of all transactions that share a `trace_id` value. 
 
-Let's refer back to our earlier example of the web application consisting of these high-level components:
+**Transactions**
 
-- Frontend (Single-Page Application)
-- Backend
-- Background Queue
-- Notification Job
+Transactions share most of their properties (start and end time, tags, and so forth) with their root spans, so the same options described below for spans are available in transactions, and setting them in either place is equivalent.
 
-A trace [propagates]({%- link _documentation/performance/performance-glossary.md -%}#propagation) first from the frontend, then to the backend, and later to the background queue or notification job. Collected [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) from each component are sent back to Sentry asynchronously and independently. Instrumented spans received from one component aren't forwarded to the next component. If a span appears to be missing a parent span, it could be an [**orphan span**]({%- link _documentation/performance/performance-glossary.md -%}#orphan-spans).
+Transactions also have one additional property not included in spans, called `transaction_name`, which is used in the UI to identify the transaction. Common examples of `transaction_name` values include endpoint paths (like `/store/checkout/` or `api/v2/users/&lt;user_id&gt;/`) for backend request transactions, task names (like `data.cleanup.delete_inactive_users`) for cron job transactions, and URLs (like `https://docs.sentry.io/performance/distributed-tracing/`) for page-load transactions.
```

Comment:
This is good and makes it more clear, we need to update our SDKs so it's called `transaction_name`.

Reply:
Yup, it's on my list! 🙂